util/translate_fight.py

```diff
@@ -192,22 +202,23 @@ def main(args):
     output_str = json.dumps(output, ensure_ascii=False, indent=2, sort_keys=True)
 
     # hacky reformatting: single quotes, and remove quotes
-    lines = []
-    headers = ['timelineReplace', 'locale', 'replaceSync', 'replaceText', 'locale']
-    for line in output_str.splitlines():
-        # add trailing commas
-        line = re.sub(r"\"\s*$", "\",", line)
-        line = re.sub(r"]\s*$", "],", line)
-        line = re.sub(r"}\s*$", "},", line)
-
-        # replace all quotes on headers
-        for header in headers:
-            if line.find('"' + header + '":') != -1:
-                line = line.replace('"', '', 2)
-        # replace double with single quotes on any line without apostrophes.
-        if line.find("'") == -1:
-            line = line.replace('"', "'")
-        lines.append(line)
+    # lines = []
```

Comment:
Could you remove all of this instead of commenting it out?  I verified that check_translation.py can import this without formatting just fine.

Reply:
yes of course https://github.com/quisquous/cactbot/pull/424/commits/1894ed8b9a50ee639efb8fc6ae1370c220fb389f